src/open_inwoner/accounts/middleware.py

```diff
@@ -29,15 +29,16 @@ def __call__(self, request):
             # If the user is currently not editing their information, but it is required
             # redirect to that view.
 
-            # DigiD can be disabled, in which case the digid app isn't available
             digid_logout = "/digid/logout/"
-            try:
-                digid_logout = reverse("digid:logout")
-            except:  # nosec
-                pass
+            digid_slo_redirect = "/digid/slo/redirect/"
```

Comment:
These URL's need to use `reverse()` (eg: reverse('digid:logout') and reverse('digid:slo-redirect'), but in a try/except for a NoReverseMatch, in case the digid app is not loaded (like in development). 

Note the original code did this but was slightly messy from some earlier hotfix.